test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs

```diff
@@ -3434,6 +3473,21 @@ public override AuthenticatedEncryptionResult Encrypt(byte[] plaintext, byte[] a
             return new AuthenticatedEncryptionResult(Key, ciphertext, iv, authenticationTag); 
         }
     }
+
+    public class DerivedJsonWebTokenHandler : JsonWebTokenHandler
+    {
+        /// <summary>
+        /// Creates a <see cref="ClaimsIdentity"/> from a <see cref="JsonWebToken"/>.
+        /// </summary>
+        /// <param name="jwtToken">The <see cref="JsonWebToken"/> to use as a <see cref="Claim"/> source.</param>
+        /// <param name="validationParameters">Contains parameters for validating the token.</param>
+        /// <param name="issuer">Specifies the issuer for the <see cref="ClaimsIdentity"/>.</param>
+        /// <returns></returns>
```

Comment:
>returns [](http://example.com/codeflow?start=13&length=7)

nit: <returns>A <see cref="ClaimsIdentity"/> containing the <see cref="JsonWebToken.Claims"/>.</returns>